tooling/cli/lib/deploy.js

```diff
@@ -140,12 +145,52 @@ process.on('message', message => {
   console.log('>>> Deploy process receive message', message);
 });
 
+async function maybeRenameEnvFile() {
+  // temporarily swap env files if it's a dev build.
+  // the CF Worker still loads an env file into process.env if it's present
+  // so we could end up with a project built with .env.development running
+  // w/ vars passed from .env instead.
+  if (isDevelopmentBuild()) {
+    await swapEnvFiles('.env', '.env.development', '.env.backup');
+  }
+
+  return;
+}
+
+async function maybeResetEnvFile() {
+  if (isDevelopmentBuild()) {
+    await swapEnvFiles('.env', '.env.backup', '.env.development');
+  }
+
+  return;
+}
+
+async function swapEnvFiles(target, source, newFilePath) {
+  const targetPath = path.join(OP_DIR, target);
+  const sourcePath = path.join(OP_DIR, source);
+
+  const hasTarget = await getFileContents(targetPath);
+  const hasSource = await getFileContents(sourcePath);
+
+  if (hasTarget && hasSource) {
+    const newPath = path.join(OP_DIR, newFilePath);
+    await rename(target, newPath);
```

Comment:
should this be:

```suggestion
    await rename(targetPath, newPath);
```

?

Reply:
Funnily enough it worked locally w/ either. Updated for consistency though, wouldn't be surprised to find out it's local configuration/link thing.